### PR TITLE
URL Input Working

### PR DESCRIPTION
**Pull request by mstephen19**
Fixed URL input broken, fixed issues based on the comments left in my previous pull request.

### CODE REVIEW

**Comment by metalwarrior665** [comment]
Thanks, I don't have a deep understanding of this actor. @pocesar perhaps knows more and we can merge it?

File: INPUT_SCHEMA.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
             "title": "Input URL",
             "type": "array",
             "description": "(Required if you don't use 'Search Queries') Here you can provide a list of search URLs.",
-            "editor": "requestListSources"
+            "editor": "requestListSources",
+            "default": "https://www.google.com/search?q=shoes&tbm=shop&sa=X"
```
Comment (pocesar) [inline INPUT_SCHEMA.json]: should be an array
```suggestion
            "prefill": [{ "url": "https://www.google.com/search?q=shoes&tbm=shop&sa=X" }]
```

**Review by pocesar — COMMENTED** [review]
thanks @mstephen19, make sure to use `apify vis` when developing locally, so you can lint the INPUT_SCHEMA. 

if you have an example run on your account with your fixes, put it here please

File: INPUT_SCHEMA.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
             "title": "Input URL",
             "type": "array",
             "description": "(Required if you don't use 'Search Queries') Here you can provide a list of search URLs.",
-            "editor": "requestListSources"
+            "editor": "requestListSources",
+            "default": "https://www.google.com/search?q=shoes&tbm=shop&sa=X",
```
Comment (metalwarrior665) [inline INPUT_SCHEMA.json:19]: You need to remove this, it is still crashing there

**Pull request merged**